src/lightning/pytorch/loggers/logger.py

```diff
@@ -120,25 +120,50 @@ def merge_dicts(  # pragma: no cover
         >>> d1 = {'a': 1.7, 'b': 2.0, 'c': 1, 'd': {'d1': 1, 'd3': 3}}
         >>> d2 = {'a': 1.1, 'b': 2.2, 'v': 1, 'd': {'d1': 2, 'd2': 3}}
         >>> d3 = {'a': 1.1, 'v': 2.3, 'd': {'d3': 3, 'd4': {'d5': 1}}}
-        >>> dflt_func = min
-        >>> agg_funcs = {'a': np.mean, 'v': max, 'd': {'d1': sum}}
+        >>> dflt_func = torch.min
+        >>> agg_funcs = {'a': torch.mean, 'v': torch.max, 'd': {'d1': torch.sum}}
```

Comment:
`max` and `sum` seemed to be Python functions and not numpy

Reply:
Committed the changes